docs/index.md

```diff
@@ -63,7 +63,7 @@ All workspace services can be deployed to all workspaces.
 
 ## User Resource
 
-A user resource is a resource that is only available to a particular researcher. For example a Guacamole VM.
```

Comment:
It is kind of weird that we call it a Guacamole VM and Guacamole Workspace Service. Few knows Guacamole and it is an implementation detail. What is a Guacamole VM?
It should properly be renamed to Virtual Desktop Workspace Service?

@marrobi @christoferlof wdut?

Reply:
I'll raise this a discussion